### ISSUE TITLE
Problem with '.' in path names

### ISSUE THREAD

**Issue by mrpink76** [nushell/nushell#1542]
**Describe the bug**
I can not cd into a directory which starts with a digit.

**To Reproduce**
```
C:\git> cd 2.Home\
error: cd unexpected .
- shell:1:10
1 | cd 2.Home\
  |     ^ unexpected argument (try cd -h)
```

I also can also not create a directory with a name starting with a digit without writing the name within quotes.

**Configuration (please complete the following information):**
 - OS: Windows 10
 - Version: nushell 0.12.0

**Comment by thegedge** [comment]
Duplicate of https://github.com/nushell/nushell/issues/1465

I'm fairly confident this is the same issue. Feel free to subscribe to it, and if this is still a problem after #1465 gets fixed, feel free to reopen 🙂
